tests/layouts/CoreLayout.spec.js

```diff
@@ -34,11 +34,4 @@ describe('(Layout) Core', function () {
     expect(_component.type).to.equal('div')
   })
 
-  it('Should render a child component.', function () {
```

Comment:
exactly... do you want me to revert this commit to leave this as component class?